pkgs/top-level/python-packages.nix

```diff
@@ -9258,7 +9258,10 @@ self: super: with self; {
 
   piccata = callPackage ../development/python-modules/piccata { };
 
-  pick = callPackage ../development/python-modules/pick { };
+  pygame = callPackage ../development/python-modules/pygame {
+    inherit (pkgs.darwin.apple_sdk.frameworks) AppKit;
+    SDL2_image = pkgs.SDL2_image_2_0_5;
+  };
```

Comment:
Please fix this part.

Reply:
This wrongfully replaces the `pick` attribute.

Reply:
Bad rebased on my end